app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -43,9 +44,13 @@ object TradeJourneyStatusesViewModel {
       if (optJourney.isEmpty) None else optJourney.head.completedState
```

Comment:
let's not use head, as list may be empty. even though you checked isEmpty, the code may change and the head will stay then. it's better to use map / haedOption etc.